Tests/Unit/Controller/TeaControllerTest.php

```diff
@@ -36,14 +36,17 @@ class TeaControllerTest extends UnitTestCase
 
     protected function setUp(): void
     {
-        $this->subject = new TeaController();
+        parent::setUp();
+
+        // We need to create an accessible mock in order to be able to set the protected `view`.
+        $this->subject = $this->getAccessibleMock(TeaController::class, ['redirect', 'forward']);
 
         $this->viewProphecy = $this->prophesize(TemplateView::class);
         $view = $this->viewProphecy->reveal();
-        $this->inject($this->subject, 'view', $view);
+        $this->subject->_set('view', $view);
 
         $this->teaRepositoryProphecy = $this->prophesize(TeaRepository::class);
-        /** @var TeaRepository&ProphecySubjectInterface $teaRepository */
+        /** @var TeaRepository|ProphecySubjectInterface $teaRepository */
```

Comment:
We need to undo this change. (PHP-CS-Fixer complaining about this is a bug avoided by #371.)